{{cookiecutter.project_slug}}/config/settings/local.py

```diff
@@ -27,6 +27,12 @@
 # Note: This key only used for development and testing.
 SECRET_KEY = env('DJANGO_SECRET_KEY', default='CHANGEME!!!')
 
+# SITE CONFIGURATION
+# ------------------------------------------------------------------------------
+# Hosts/domain names that are valid for this site
+# See https://docs.djangoproject.com/en/1.6/ref/settings/#allowed-hosts
+ALLOWED_HOSTS=env.list('DJANGO_ALLOWED_HOSTS', default=['*'])
```

Comment:
Why is the default `['*']`? According to the doc:
> When DEBUG is `True` and ALLOWED_HOSTS is empty, the host is validated against `['localhost', '127.0.0.1', '[::1]']`.

Shouldn't be the default value empty list?